src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/util/OcrFormValidationHelper.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.util;
+
+import org.apache.commons.validator.routines.EmailValidator;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.OcrDataField;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+public final class OcrFormValidationHelper {
+
+    private OcrFormValidationHelper() {
+        // util class
+    }
+
+    public static List<String> getOcrFieldNames(List<OcrDataField> ocrData) {
+        return ocrData
+            .stream()
+            .map(field -> field.key)
+            .collect(toList());
```

Comment:
Fields might have duplicated keys, so best to invoke `.distinct()`

Reply:
... or just operate on sets

Reply:
I would imagine if there are duplicate keys then it should. result in error as Exela would need to correct and resend it.
We can't ignore it as we don't know which value is correct and it can't be a warning I think as the caseworker won't be able to remove any keys from CCD UI.

Reply:
Good point @lgonczar!
I missed the duplicate keys validation scenario. I'll update the code to fail the ocr data validation in case of duplicate keys.